<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1689</width>
    <height>1100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>7</y>
      <width>1800</width>
      <height>1031</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="mainTask2">
     <attribute name="title">
      <string>Задача 11</string>
     </attribute>
     <widget class="QLabel" name="main2IntroText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Численно решить задачу Коши для системы ОДУ 2-го порядка.&lt;br/&gt;Использовать метод Рунге-Кутта 4 порядка.&lt;br/&gt;Параметр 'L' должен быть положительным.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="main2Picture_Equation">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>231</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Resources/DE3.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="main2SolveButton">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>310</y>
        <width>141</width>
        <height>55</height>
       </rect>
      </property>
      <property name="text">
       <string>Решить!</string>
      </property>
     </widget>
     <widget class="QLabel" name="main2Picture_x0">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>162</y>
        <width>30</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Resources/x_0.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="main2LineEdit_u0">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>200</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="main2Picture_nmax">
      <property name="geometry">
       <rect>
        <x>165</x>
        <y>250</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Resources/Nmax.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="main2LineEdit_h">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="main2LineEdit_nmax">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>250</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="main2Picture_v_eps">
      <property name="geometry">
       <rect>
        <x>194</x>
        <y>281</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Resources/v_eps.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="main2Picture_x_eps">
      <property name="geometry">
       <rect>
        <x>193</x>
        <y>191</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Resources/x_eps.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="main2LineEdit_v_eps">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>280</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="main2Picture_h">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>275</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Resources/h.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="main2ShowSolnInfoButton">
      <property name="geometry">
       <rect>
        <x>191</x>
        <y>310</y>
        <width>141</width>
        <height>55</height>
       </rect>
      </property>
      <property name="text">
       <string>Посмотреть
результаты</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="main2LineEdit_xmax">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="main2LineEdit_x0">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="main2Picture_xmax">
      <property name="geometry">
       <rect>
        <x>165</x>
        <y>161</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Resources/x_max.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="main2LineEdit_x_eps">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>190</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="main2Picture_u0">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>203</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Resources/u_0.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="main2LineEdit_u0_dash">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="main2Picture_u0_dash">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>234</y>
        <width>32</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Resources/u'0.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="main2LineEdit_L">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>220</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="main2Picture_a">
      <property name="geometry">
       <rect>
        <x>204</x>
        <y>220</y>
        <width>25</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Resources/a.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="main2Table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>511</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="main2Plot1Background" native="true">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>70</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="main2Plot3Background" native="true">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>100</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="main2Plot1" native="true">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>10</y>
        <width>681</width>
        <height>841</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="main2Plot2" native="true">
      <property name="geometry">
       <rect>
        <x>1220</x>
        <y>10</y>
        <width>601</width>
        <height>415</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="main2Plot2Background" native="true">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>70</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="main2Plot3" native="true">
      <property name="geometry">
       <rect>
        <x>1220</x>
        <y>440</y>
        <width>571</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="introTextButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Справка</string>
      </property>
     </widget>
     <zorder>main2Plot3Background</zorder>
     <zorder>main2Plot2Background</zorder>
     <zorder>main2Plot1Background</zorder>
     <zorder>main2IntroText</zorder>
     <zorder>main2Picture_Equation</zorder>
     <zorder>main2SolveButton</zorder>
     <zorder>main2Picture_x0</zorder>
     <zorder>main2LineEdit_u0</zorder>
     <zorder>main2Picture_nmax</zorder>
     <zorder>main2LineEdit_h</zorder>
     <zorder>main2LineEdit_nmax</zorder>
     <zorder>main2Picture_v_eps</zorder>
     <zorder>main2Picture_x_eps</zorder>
     <zorder>main2LineEdit_v_eps</zorder>
     <zorder>main2Picture_h</zorder>
     <zorder>main2ShowSolnInfoButton</zorder>
     <zorder>main2LineEdit_xmax</zorder>
     <zorder>main2LineEdit_x0</zorder>
     <zorder>main2Picture_xmax</zorder>
     <zorder>main2LineEdit_x_eps</zorder>
     <zorder>main2Picture_u0</zorder>
     <zorder>main2LineEdit_u0_dash</zorder>
     <zorder>main2Picture_u0_dash</zorder>
     <zorder>main2LineEdit_L</zorder>
     <zorder>main2Picture_a</zorder>
     <zorder>main2Table</zorder>
     <zorder>main2Plot1</zorder>
     <zorder>main2Plot2</zorder>
     <zorder>main2Plot3</zorder>
     <zorder>introTextButton</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1689</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
